.github/workflows/update-dockerfile.yaml

```diff
@@ -76,12 +76,24 @@ jobs:
       - name: Create or Update Pull Request
         if: steps.check_changes.outputs.changes_detected == 'true'
         run: |
-          gh pr create \
-            -B master \
-            -H update-dockerfile \
-            --title "Update Dockerfile with latest Docker and Buildx versions" \
-            --body "Updates the Dockerfile to use the latest versions:
-              - Docker: ${{ steps.get_docker_version.outputs.docker_version }}
-              - Buildx: ${{ steps.get_buildx_version.outputs.buildx_version }}"
+          # Check if a pull request for the branch "update-dockerfile" already exists
+          pr_number=$(gh pr list --head update-dockerfile --state open --json number --jq '.[].number')
```

Comment:
it might be nice to pull-up branch-name to a shared variable (I think doing this via and env-var would be the idiomatic way to do that in github-actions)
(not strictly related to the scope of this pullrequest, admittedly)